test/cases/backdrop.out.css

```diff
@@ -0,0 +1,6 @@
+::-ms-backdrop {
+  color: green;
+}
```

Comment:
I suspect that IE hasn't implemented this pseudo-element either, which I'm afraid requires further investigation and testing.